arangod/Utils/ExecContext.h

```diff
@@ -159,6 +163,25 @@ struct ExecContextScope {
  private:
   ExecContext const* _old;
 };
+  
+struct ExecContextSuperuserScope {
+  explicit ExecContextSuperuserScope()
+  : _old(ExecContext::CURRENT) {
+    ExecContext::CURRENT = &ExecContext::Superuser;
+  }
+  
+  ExecContextSuperuserScope(bool cond) : _old(ExecContext::CURRENT) {
```

Comment:
can we make this ctor `explicit`?
```suggestion
  explicit ExecContextSuperuserScope(bool cond) : _old(ExecContext::CURRENT) {
```